Helper/Data.php

```diff
@@ -336,9 +338,9 @@ public function callApi($params, $payment, $type='transactions')
         $params = $paramsObj->getParams();
         $paramsString = $this->convertToCURLString($params);
 
-        //$this->writeLog('Parameters being sent to API (/'.$type.'): '. var_export($params, true));
+        $this->writeLog('Parameters being sent to API (/'.$type.'): '. var_export($params, true));
```

Comment:
I'm almost sure that you don't need the var_export, and this can be passed as the second argument of writeLog method.

Reply:
@r-martins there's only 1 parameter in writeLog function. var_export is needed since $param was concatenated into a string, Otherwise the log will look like this:

> [2018-11-13 08:02:38] main.DEBUG: Parameters being sent to API (/transactions):  [] []